Tests/MapboxNavigationTests/RouteControllerSnapshotTests.swift

```diff
@@ -49,19 +47,19 @@ class RouteControllerSnapshotTests: FBSnapshotTestCase {
         let routeController = RouteController(along: route, routeIndex: 0, options: equivalentRouteOptions, dataSource: self)
         locationManager.delegate = routeController
         
-        var snappedLocations = [CLLocation]()
+        var snappedLocations = Set<CLLocation>()
         
         while snappedLocations.count < locationManager.locations.count {
             locationManager.tick()
-            snappedLocations.append(routeController.location!)
+            snappedLocations.insert(routeController.location!)
         }
         
-        let view = NavigationPlotter(frame: CGRect(origin: .zero, size: CGSize(width: 1000, height: 1000)))
-        view.routePlotters = [RoutePlotter(route: route)]
-        view.locationPlotters = [LocationPlotter(locations: locations, color: #colorLiteral(red: 0.9254902005, green: 0.2352941185, blue: 0.1019607857, alpha: 0.5043463908), drawIndexesAsText: true),
-                                 LocationPlotter(locations: snappedLocations, color: #colorLiteral(red: 0.4666666687, green: 0.7647058964, blue: 0.2666666806, alpha: 0.3969795335), drawIndexesAsText: true)]
+        let expectedLocations = Set<CLLocation>()
+        coordinates.forEach{
+            expectedLocations.insert(CLLocation(latitude: $0.coordinate.latitude, longitude: $0.coordinate.longitude))
+        }
```

Comment:
This would be more succinctly written as:

```suggestion
        expectedLocations = coordinates.map {
            CLLocation(latitude: $0.coordinate.latitude, longitude: $0.coordinate.longitude)
        }
```

Reply:
Thanks for the suggestion, I'll replace this part.